app/views/users/edit.html.erb

```diff
@@ -0,0 +1,41 @@
+
+<h1> Sign Up </h1>
+
+<%# <form action="/users" method="post" accept-charset="UTF-8"> %>
+  <%# <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token ">%>
+ <%# <label for="username">User name:</label> %>
+ <%# <input type="text" id="username" name="user[username]"><br><br> %>
+ <%# <label for="email">E-mail:</label> %>
+ <%# <input type="text" id="email" name="user[email]"><br><br> %>
+ <%# <label for="password">Password:</label> %>
+ <%# <input type="password" id="password" name="user[password]"><br><br> %>
+ <%# <input type="submit" value="Submit"> %>
+<%# </form> %>
+
+
+<%# <%= form_tag("/users", method: "post") do %>
+<%# <div> %>
+  <%# <%= label_tag(:username, "Username") %>
+  <%# <%= text_field_tag(:username) %>
+  <%# </div> %>
+  <%# <div> %>
+  <%# <%= label_tag(:email, "Email") %>
+  <%# <%= text_field_tag(:email) %>
```

Comment:
- [ ] Use `email_field_tag` instead of `text_field_tag` for the email. That way rails will know to compile it as `<input type="email">`